backend/models/user.js

```diff
@@ -39,6 +39,10 @@ const UserSchema = new Schema({
     type: String,
     required: false,
   },
+  grad_quarter: {
+    type: String,
```

Comment:
```suggestion
    type: String,
    enum: ["Winter", "Spring", "Summer", "Fall"],
```

This would help prevent any unexpected values from being saved to the database. Haven't tested this, but it should work based on [this documentation](https://mongoosejs.com/docs/validation.html).